server/routes/components.ts

```diff
@@ -117,17 +117,23 @@ export default function routes({ serviceCatalogueService, redisService }: Servic
 
     if (environments[0].ip_allow_list && environments[0].ip_allow_list_enabled) {
       const ipAllowListFiles = Object.keys(environments[0].ip_allow_list)
-      allowList.set('groups', [])
 
       ipAllowListFiles.forEach(fileName => {
         if (Object.keys(environments[0].ip_allow_list[fileName]).length > 0) {
```

Comment:
Can probably ditch the `if` now as the `foreach` simply won't do anything if the length is 0. Or should behave that way